.eslintrc.js

```diff
@@ -91,6 +91,31 @@ module.exports = {
         "aksel-local/comment-check": ["error"],
       },
     },
+    {
+      files: [
+        "aksel.nav.no/website/components/website-modules/examples/__parts*/*.tsx",
+      ],
+      plugins: ["aksel-local"],
+      rules: {
+        "aksel-local/import-check": ["error"], // Only allow imports from @navikt and react
+      },
+    },
+    {
+      files: [
+        "aksel.nav.no/website/components/website-modules/examples/__parts-inline/*.tsx",
```

Comment:
```suggestion
        "**/examples/__parts*/*.tsx",
      ],
      plugins: ["aksel-local"],
      rules: {
        "aksel-local/import-check": ["error"], // Only allow imports from @navikt and react
      },
    },
    {
      files: [
        "**/examples/__parts-inline/*.tsx",
```
Would this suffice when scoping the ruleset?